README.rst

```diff
@@ -179,6 +179,20 @@ field. The currently supported configuration options are:
     Setting this property to true will force the harvester to gather all remote
     packages regardless of the modification date. Default is False.
 
+*   organizations_filter_include: This configuration option allows you to specify
+    a list of remote organization short names. If this property has a
```

Comment:
I don't think we've referred to "short name" before. How about calling it "organization name" but give an example: "(e.g. "arkansas-gov" is the name for organization http://catalog.data.gov/organization/arkansas-gov )"?